internal/controller/nonadminbackup_controller_test.go

```diff
@@ -25,7 +25,7 @@ import (
 
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
-	"github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
```

Comment:
I like this change, but lets enforce it

In `.golangci.yml` please do the following change
```diff
@@ -47,6 +47,11 @@ linters-settings:
       limitations under the License.
   govet:
     enable-all: true
+  importas:
+    no-unaliased: true
+    alias:
+      - pkg: github.com/vmware-tanzu/velero/pkg/apis/velero/v1
+        alias: velerov1
   misspell:
     locale: US
   nakedret:
@@ -95,6 +100,7 @@ linters:
     - gosec
     - gosimple
     - govet
+    - importas
     - ineffassign
     - misspell
     - nakedret
```

This will enforce that every place that imports velero v1 package in the project, uses the `velerov1` alias